srcpkgs/python3-libtmux/template

```diff
@@ -10,7 +10,7 @@ maintainer="Alexander Egorenkov <egorenar-dev@posteo.net>"
 license="MIT"
 homepage="https://github.com/tmux-python/libtmux"
 distfiles="${PYPI_SITE}/l/libtmux/libtmux-${version}.tar.gz"
-checksum=d82cf391097eb69d784d889d482bb99284b984aa6225276a3dc1af8c1460dd3c
+checksum=c40959c5ae9e7add705a3c4e60365667e65ab45835a9e663692e428e1d7e42e2
 # Disable check which uses pip to pull external python packages
 make_check=no
```

Comment:
The reason is wrong and at the time it was added it was probably just a missing dependency.
```suggestion
checkdepends="python3-pytest tmux"
# fixture 'config_file' not found
make_check=no
```

Reply:
thanks! fixed.